### ISSUE TITLE
upgrade gadgetron to 3.17 or later

### ISSUE THREAD

**Issue by KrisThielemans** [SyneRBI/SIRF-SuperBuild#305]
see https://github.com/CCPPETMR/CCPPETMR_VM/issues/141 for possible issues. might need changes to travis and docker dependencies

**Comment by KrisThielemans** [comment]
and wiki

**Comment by KrisThielemans** [comment]
update `version_config` but need to check with VM as well

**Comment by KrisThielemans** [comment]
@evgueni-ovtchinnikov can you check this one please? It's the last major change before 2.3 I believe.

**Comment by evgueni-ovtchinnikov** [comment]
Build fails with
```
CMake Error at /usr/local/share/cmake-3.13/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find LAPACKE (missing: LAPACKE_INCLUDE_DIR LAPACKE_LIBRARIES)
Call Stack (most recent call first):
  /usr/local/share/cmake-3.13/Modules/FindPackageHandleStandardArgs.cmake:378 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindLAPACKE.cmake:19 (find_package_handle_standard_args)
  CMakeLists.txt:300 (find_package)
```

**Comment by rijobro** [comment]
You'll need to install extra prerequisites, specifically LAPACKE:

https://github.com/SyneRBI/SyneRBI_VM/blob/e56b42c1e260e5457e5650afe4f1a0b99ae5b2b9/scripts/INSTALL_prerequisites_with_apt-get.sh#L21-L23

**Comment by evgueni-ovtchinnikov** [comment]
should i add it then to `INSTALL_prerequisites_with_apt-get.sh`?

**Comment by rijobro** [comment]
I noticed it was missing and added a few minutes ago: https://github.com/SyneRBI/SyneRBI_VM/commit/80e4ce94e1cad64e72b2cc7f751386321d9ab719.

**Comment by KrisThielemans** [comment]
will need to be added in the docker scripts as well, as these are still independent.

**Comment by KrisThielemans** [comment]
On MacOS, @DANAJK wrote the following on Slack:

The instructions for the current course on gadgetron say:

Mac OS
Gadgetron doesn't currently build on Mac OS. You may try MacPorts or HomeBrew. However, users should have better luck with the following options:
- Software such as Parallels, VirtualBox, etc., can be used to install a Ubuntu virtual machine (VM), then install Gadgetron into that Ubuntu VM, as detailed above for the native Ubuntu installation.
- Alternatively, Docker Desktop Community Edition for Mac can be used to download and install a Gadgetron Docker container. After the Docker software installation is complete, and a network connection is available, the Gadgetron Docker container is installed by running the command:
```
docker pull gadgetron/ubuntu_2004
```
Once the container has been downloaded and installed into the user's Docker environment, the user should then be able to launch it as they would any other container, for example:
```
docker run --name gtLocal gadgetron/ubuntu_2004
```
and interact with the running container using any of Docker's standard suite of command line tools.

**Comment by rijobro** [comment]
It also doesn't look like Gadgetron will be functional on OSX any time soon, see: https://github.com/gadgetron/gadgetron/issues/822#issuecomment-650198018 (and possibly: https://github.com/gadgetron/gadgetron/issues/787#issuecomment-650199040).

**Comment by paskino** [comment]
Work is ongoing in #438

**Comment by paskino** [comment]
closed by #438
